src/plugins/dashboard/public/application/embeddable/dashboard_container_factory.tsx

```diff
@@ -85,11 +85,15 @@ export class DashboardContainerFactoryDefinition
       ControlGroupOutput,
       ControlGroupContainer
     >(CONTROL_GROUP_TYPE);
+    const { filters, query, timeRange, viewMode } = initialInput;
```

Comment:
nit: why not also destructure `id` or `controlGroupInput` which are also used below?

Reply:
Done!